apps/frontend/src/renderer/components/gitlab-issues/components/InvestigationDialog.tsx

```
@@ -0,0 +1,234 @@
+                  >
+                    {selectedNoteIds.length === notes.length ? 'Deselect All' : 'Select All'}
+                  </Button>
+                </div>
+                <ScrollArea className="flex-1 min-h-0 border rounded-md">
+                  <div className="p-2 space-y-2">
+                    {notes.map((note) => (
+                      <div
+                        key={note.id}
+                        className="flex gap-3 p-3 rounded-lg border border-border bg-card hover:bg-accent/50 transition-colors cursor-pointer"
+                        onClick={() => toggleNote(note.id)}
+                      >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)